src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -374,1065 +349,576 @@
-          isCorrectionPending: (header.isCorrectionPending || false),
-          comments: this.flattenAndSortComments(header.comments)
-        }
-
-        // Apply additional properties for Staff Only Filings
-        if (this.isStaffFiling(filingType)) {
-          let baseFiling
-          switch (filingType) {
-            case FilingTypes.REGISTRARS_NOTATION: baseFiling = filing.registrarsNotation
-              break
-            case FilingTypes.REGISTRARS_ORDER: baseFiling = filing.registrarsOrder
-              break
-            case FilingTypes.COURT_ORDER: baseFiling = filing.courtOrder
-              break
-          }
-
-          item.notationOrOrder = baseFiling?.orderDetails
-          item.fileNumber = baseFiling?.fileNumber
-          item.planOfArrangement = baseFiling?.effectOfOrder ? 'Pursuant to a Plan of Arrangement' : ''
 
-          item.filingAuthor = this.isRoleStaff ? header.submitter : 'Registry Staff'
-        }
-
-        // add receipt
-        if (header.paymentToken) {
-          item.documents.push({
-            type: this.DOCUMENT_TYPE_RECEIPT,
-            corpName: this.entityName || this.getCorpTypeNumberedDescription(this.entityType),
-            filingDateTime,
-            paymentToken: header.paymentToken,
-            title: 'Receipt',
-            filename: `${this.entityIncNo} - Receipt - ${filingDate}.pdf`
-          })
-        }
-
-        this.historyItems.push(item)
-      } else {
-        // eslint-disable-next-line no-console
-        console.log('ERROR - missing section in filing =', filing)
+      // special handling for paper-only filings
+      if (filing.availableOnPaperOnly) {
+        // clear URL so we don't try to load documents
+        item.documentsLink = null
       }
-    },
-
-    /** Whether this filing's effective date/time is in the past. */
-    isEffectiveDatePast (filing: FilingIF): boolean {
-      const header = filing?.header as FilingHeaderIF
-      if (header?.effectiveDate) {
-        // NB: these are both in UTC
-        const effectiveDateTime = new Date(header.effectiveDate)
-        const now = new Date()
-        if (effectiveDateTime <= now) {
-          return true
-        }
-      }
-      return false
-    },
-
-    /** Whether this filing's effective date/time is in the future. */
-    isEffectiveDateFuture (filing: FilingIF): boolean {
-      const header = filing?.header as FilingHeaderIF
-      if (header?.effectiveDate) {
-        // NB: these are both in UTC
-        const effectiveDateTime = new Date(header.effectiveDate)
-        const now = new Date()
-        if (effectiveDateTime > now) {
-          return true
-        }
-      }
-      return false
-    },
 
-    /** Whether this filing(alteration) is pursuant to a plan of arrangement  */
-    isPlanOfArrangement (courtOrder: CourtOrderIF): boolean {
-      if (!courtOrder || !courtOrder.effectOfOrder) {
-        return false
+      // add properties for correction filings
```

Comment:
line 518 and 524 comments should be swapped?

Reply:
I think this is actually correct:
- a Correction Filing has the id of the filing that it corrects
- A Corrected Filing has the id of the filing that corrects it

Reply:
ok 👍